packages/block-library/src/image/edit.js

```diff
@@ -423,7 +425,7 @@ class ImageEdit extends Component {
 				<Fragment>
 					{ controls }
 					<MediaPlaceholder
-						icon="format-image"
+						icon=<BlockIcon icon={ icon } />
```

Comment:
Can it follow other definitions of props?:
```suggestion
						icon={ <BlockIcon icon={ icon } /> }
```